vmm/src/memory_manager.rs

```diff
@@ -1543,22 +1554,44 @@ impl Snapshottable for MemoryManager {
         let mut memory_manager_snapshot = Snapshot::new(MEMORY_MANAGER_SNAPSHOT_ID);
         let guest_memory = self.guest_memory.memory();
 
-        let mut memory_regions: Vec<MemoryRegion> = Vec::with_capacity(10);
+        let mut memory_regions: Vec<MemoryRegion> = Vec::new();
```

Comment:
That seems to be fixing a limitation that was not needed, right?

Reply:
Yes. Do you want this to be part of a separate commit?

Reply:
No that's ok.